paper-checkbox.html

```diff
@@ -64,9 +64,9 @@
   
     <paper-ripple id="ink" class="circle recenteringTouch" checked?="{{!checked}}"></paper-ripple>
      
-    <div id="checkbox">
-      <div id="checkmark" on-animationend="{{checkboxAnimationEnd}}" on-webkitAnimationEnd="{{checkboxAnimationEnd}}"></div>
-    </div>
+  	<div id="checkbox" class="{{ {checked: checked} | tokenList }}">
+  		<div id="checkmark" class="{{ {hidden: !checked} | tokenList }}" on-animationend="{{}}" on-webkitanimationend="{{}}"></div>

```

Comment:
`on-animationend` and `on-webkitanimationend` can both be removed here since they aren't doing anything.